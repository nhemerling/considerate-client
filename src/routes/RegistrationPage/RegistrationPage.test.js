import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import RegistrationPage from './RegistrationPage';

describe('RegistrationPage component', () => {
  it('renders a RegistrationPage component', () => {
    const wrapper = shallow(<RegistrationPage />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
