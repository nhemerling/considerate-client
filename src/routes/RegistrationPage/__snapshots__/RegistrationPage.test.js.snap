// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RegistrationPage component renders a RegistrationPage component 1`] = `
<section
  className="RegistrationPage"
>
  <h2>
    Register
  </h2>
  <RegistrationForm
    onRegistrationSuccess={[Function]}
  />
</section>
`;
